cluster/examples/kubernetes/ceph/filesystem-test.yaml

```diff
@@ -18,7 +18,8 @@ spec:
       replicated:
         size: 1
         requireSafeReplicaSize: false
-      compressionMode: none
```

Comment:
While we're at it, could you just remove the compression examples from all the test yamls? The idea of the test yamls is to be small and simple for very basic test clusters and we wouldn't ever need compression there anyway.

Reply:
Done